### PR TITLE
Update AudioWorklet basic examples

### LINKED ISSUE #119: Update AudioWorklet basic examples

### PR DESCRIPTION
**Pull request by hoch**
Fixes #119.

### CODE REVIEW

File: audio-worklet/basic/audio-param-modulation.html [hunk 1]
```diff
@@ -13,23 +13,23 @@
...
 
-        // Connect the osicllator to 'amplitude' AudioParam.
-        paramAmp = noiseGenerator.parameters.get('amplitude');
-        modulator.connect(paramAmp);
-        
-        modulator.frequency.value = 1;
+        // Connect the oscillator to 'amplitude' AudioParam.
+        let paramAmp = noiseGenerator.parameters.get('amplitude');
+        modulator.connect(modGain).connect(paramAmp);
+        // modulator.connect(noiseGenerator);
```
Comment (rtoy) [inline audio-worklet/basic/audio-param-modulation.html]: Remove this?

File: audio-worklet/basic/js/noise-generator.js [hunk 2]
```diff
@@ -11,12 +13,17 @@ class NoiseGenerator extends AudioWorkletProcessor {
...
-    let outputChannelData = output.getChannelData(0);
+  process(inputs, outputs, parameters) {
+    let output = outputs[0];
     let amplitude = parameters.amplitude;
-    for (let i = 0; i < 128; ++i) {
-      outputChannelData[i] = Math.random() * amplitude[i];
+    for (let channel = 0; channel < output.length; ++channel) {
+      let outputChannel = output[channel];
+      for (let i = 0; i < outputChannel.length; ++i) {
+        outputChannel[i] = Math.random() * amplitude[i];
```
Comment (rtoy) [inline audio-worklet/basic/js/noise-generator.js]: Probably already mentioned this earlier, but Math.random returns a value in the range [0, 1).  I think a noise generator should have zero mean.

File: audio-worklet/basic/js/one-pole.js [hunk 3]
```diff
@@ -1,28 +1,44 @@
...
  * @extends AudioWorkletProcessor
- *
- * Implements a simple OnePole filter. 
  */
 class OnePole extends AudioWorkletProcessor {
+  static get parameterDescriptors() {
+    return [{
+      name: 'frequency',
+      defaultValue: 250,
+      minValue: 0
```
Comment (rtoy) [inline audio-worklet/basic/js/one-pole.js]: Since we chatted about this, maybe add comment that maxValue is defaulted on purpose because we can't set it to the (unknown) Nyquist frequency?

File: resources/base.css [hunk 4]
```diff
@@ -19,7 +19,7 @@ a.link-small {
 
 ul {
   list-style: none;
-  padding-left: 26px;
+  columns: 2;
```
Comment (rtoy) [inline resources/base.css]: Why this change?

File: resources/base.css [hunk 5]
```diff
@@ -30,6 +30,11 @@ ul > li {
   font-size: 14px;
 }
 
+div.file-issue {
+  float: right;
+  font-size: 12px;
+}
```
Comment (rtoy) [inline resources/base.css]: Why this change?

File: audio-worklet/basic/js/noise-generator.js (hunk 2 above)
Comment (hoch) [reply audio-worklet/basic/js/noise-generator.js]: Oops. Fixed.

File: audio-worklet/basic/js/one-pole.js (hunk 3 above)
Comment (hoch) [reply audio-worklet/basic/js/one-pole.js]: Done.

File: resources/base.css (hunk 4 above)
Comment (hoch) [reply resources/base.css]: I rearranged the index page and this gives much nicer layout.

File: resources/base.css (hunk 5 above)
Comment (hoch) [reply resources/base.css]: Removed.
